unlock-app/src/middlewares/storageMiddleware.js

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
I think 2 unintended files made it into this commit?

Reply:
This is me updating spots where the code is now bleeding, the tldr is that the code was updated  in a way that starts to breaks subtly if Locksmith isn't available. A 404 isn't a fatal issue, but not handling a promise rejection might be

Reply:
gotcha. 👍